### ISSUE TITLE
Address Nerd Font FontPatcher issues

### ISSUE THREAD

**Issue by ktekinay** [0xType/0xProto#40]
Until Nerd Font includes 0xProto, I use their script to merge the glyphs. This works well, but I get a report of potential issues. I have no idea if any of these are worthy of action, but I thought I'd bring it to your attention.

This report starts with the issued command including switches.

[nerd-font-report.txt](https://github.com/0xType/0xProto/files/13281556/nerd-font-report.txt)

**Comment by masaakim** [comment]
@ktekinay Just to be clear, is this issue simply a reminder and you believe that no changes to 0xProto are necessary at this point?

**Comment by masaakim** [comment]
Understanding that some may hesitate to download files with unclear contents, even with a .txt extension, I’ll ensure to include that command on the page for transparency and peace of mind.

```
fontforge --script FontPatcher/font-patcher --careful --complete --mono 0xProto-Regular.otf
```

**Comment by ktekinay** [comment]
Not knowing anything about what goes into font creation, I don't know if any changes are necessary. I brought these to your attention in case any, all, or even none need action.

**Comment by masaakim** [comment]
Alright, I'll consider this issue resolved once 0xProto is incorporated into Nerd Font.
I'm not familiar with their inclusion criteria, but I don't anticipate needing to modify the 0xProto source files themselves for this.

**Comment by masaakim** [comment]
It seems that 0xProto has been included since Nerd Font v3.10 and available for download from nerdfonts.com

- https://github.com/ryanoasis/nerd-fonts/releases/tag/v3.1.0
- https://www.nerdfonts.com/font-downloads
